Fabric.Authorization.API/RemoteServices/Identity/Providers/IdentityServiceProvider.cs

```diff
@@ -52,13 +60,27 @@ public IdentityServiceProvider(IHttpClientFactory httpClientFactory, IAppConfigu
                 UserIds = userIdList
             };
 
-            var response = await httpClient.PostAsync("/api/users",
+            _logger.Debug($"Invoking Fabric.Identity endpoint {httpClient.BaseAddress}");
+
+            var response = await httpClient.PostAsync("api/users",
                 new StringContent(JsonConvert.SerializeObject(request), Encoding.UTF8, "application/json"));
 
-            return response.StatusCode != HttpStatusCode.OK
-                ? new List<UserSearchResponse>()
-                : JsonConvert.DeserializeObject<IEnumerable<UserSearchResponse>>(await response.Content
+            var results = new List<UserSearchResponse>();
+            if (response.StatusCode != HttpStatusCode.OK)
+            {
+                _logger.Error($"Response from Fabric.Identity api/users => {response.StatusCode}");
```

Comment:
Should we also log the content of the response, in case there is an error message from the remote service?